benches/bench_for_estimates.rs

```diff
@@ -0,0 +1,287 @@
+                }
+            )*
+        }
+    }
+}
+
+//make_benches!(planned, planned, {2,3,4,5,6,7,8,11,13,16,17,19,23,29,31,32,127,233});
+make_benches!(radix4, radix4, {4, 8, 16, 32, 64, 128, 256, 512, 1024, 2048, 4096, 8192, 16384, 32768, 65536, 131072, 262144, 524288, 1048576, 2097152, 4194304});
+//make_benches!(mixedradix_rx4, mixedradix_rx4, {32, 64, 128, 256, 512, 1024, 2048});
+make_benches_two_args!(mixedradix, mixedradix, {(3, 4), (3, 5), (3, 7), (3,13), (3,31), (7, 31), (23, 31), (29,31), (31, 127), (31, 233), (127, 233), (127, 1031), (1031, 2003)});
+make_benches_two_args!(goodthomas, goodthomas, {(3, 4), (3, 5), (3, 7), (3,13), (3,31), (7, 31), (23, 31), (29,31), (31, 127), (31, 233), (127, 233), (127, 1031), (1031, 2003)});
+make_benches_two_args!(mixedradixsmall, mixedradixsmall, {(3, 4), (3, 5), (3, 7), (3,13), (3,31), (7, 31), (23, 31), (29,31)});
+make_benches_two_args!(goodthomassmall, goodthomassmall, {(3, 4), (3, 5), (3, 7), (3,13), (3,31), (7, 31), (23, 31), (29,31)});
+make_benches_two_args!(planned_multi, planned_multi, {(2,1),(3,1), (4,1), (5,1), (6,1), (7,1), (8,1), (11,1), (13,1), (16,1), (17,1), (19,1), (23,1), (29,1), (31,1), (32,1)});
+make_benches_two_args!(planned_multi, planned_multi, {(2,2),(3,2), (4,2), (5,2), (6,2), (7,2), (8,2), (11,2), (13,2), (16,2), (17,2), (19,2), (23,2), (29,2), (31,2), (32,2)});
+make_benches_two_args!(planned_multi, planned_multi, {(2,3),(3,3), (4,3), (5,3), (6,3), (7,3), (8,3), (11,3), (13,3), (16,3), (17,3), (19,3), (23,3), (29,3), (31,3), (32,3)});
+make_benches_two_args!(planned_multi, planned_multi, {(2,5),(3,5), (4,5), (5,5), (6,5), (7,5), (8,5), (11,5), (13,5), (16,5), (17,5), (19,5), (23,5), (29,5), (31,5), (32,5)});
+make_benches_two_args!(planned_multi, planned_multi, {(2,8),(3,8), (4,8), (5,8), (6,8), (7,8), (8,8), (11,8), (13,8), (16,8), (17,8), (19,8), (23,8), (29,8), (31,8), (32,8)});
+make_benches_two_args!(planned_multi, planned_multi, {(2,13),(3,13), (4,13), (5,13), (6,13), (7,13), (8,13), (11,13), (13,13), (16,13), (17,13), (19,13), (23,13), (29,13), (31,13), (32,13)});
+make_benches_two_args!(planned_multi, planned_multi, {(2,21),(3,21), (4,21), (5,21), (6,21), (7,21), (8,21), (11,21), (13,21), (16,21), (17,21), (19,21), (23,21), (29,21), (31,21), (32,21)});
+make_benches_two_args!(planned_multi, planned_multi, {(2,34),(3,34), (4,34), (5,34), (6,34), (7,34), (8,34), (11,34), (13,34), (16,34), (17,34), (19,34), (23,34), (29,34), (31,34), (32,34)});
+make_benches_two_args!(planned_multi, planned_multi, {(2,55),(3,55), (4,55), (5,55), (6,55), (7,55), (8,55), (11,55), (13,55), (16,55), (17,55), (19,55), (23,55), (29,55), (31,55), (32,55)});
+make_benches_two_args!(planned_multi, planned_multi, {(2,89),(3,89), (4,89), (5,89), (6,89), (7,89), (8,89), (11,89), (13,89), (16,89), (17,89), (19,89), (23,89), (29,89), (31,89), (32,89)});
+
+make_benches_two_args!(mixinners, planned_multi, {(3, 4), (4, 3), (3, 5), (5, 3), (3, 7), (7, 3),  (3,13), (13,3), (3,31), (31,3)});
```

Comment:
What's the difference between the "mixinners" benchmarks and the "planned_multi" benchmarks up above? It looks like the only arguments to mixinners are prime - is that it? It might be worth adding a comment to each block of benchmark explaining what they're for.

Reply:
I'll add comments.
The "planned_multi" are used for estimating the butterfly costs.
The mixinners are there to measure the same combinations of length and repeats as for the inners when measuring the mixed radix algos. The mixed radixes are tricky because the overhead of transposing and twiddling is quite small compared to the inner ffts. I first tried to use estimated values from fitting the butterflys, but that gave too much noise. Doing really the same much better results.